apps/ejabberd/src/mod_mam_odbc_async_pool_writer.erl

```diff
@@ -60,6 +60,13 @@ worker_prefix() ->
 worker_count(_Host) ->
```

Comment:
It is actually tricky , because it is used by `stop_workers`, I didn't pass `PoolSize`  there, because `stop(Host)` have no `Opts` argument, should I get the pool size somehow differently to stop the module? I can it is possible.